runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -1073,8 +1288,7 @@ void chpl_comm_wait_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles)
 }
 
 
-int chpl_comm_try_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles)
-{
+int chpl_comm_try_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles) {
```

Comment:
Not important, but I keep wanting to propose that me and you come up with a style guide we like for the runtime and start running clang-format to enforce it.

Reply:
That would be great!  That particular change is because I lifted this code from comm=ugni where I used an idiosyncratic style for outermost curly braces that I have rued every since.  Here I'm undoing it for comm=ofi so it matches most other runtime code.